pkg/services/reputation/local/route/deps.go

```diff
@@ -18,6 +18,17 @@ type ServerInfo interface {
 	Address() string
 }
 
+// Builder groups methods to route values in the network.
+type Builder interface {
+	// NextStage must return next group of route points for passed epoch
+	// based on the passed route.
+	//
+	// Empty passed list means being at the starting point of the route.
+	//
+	// Must return empty list and no error if the endpoint of the route is reached.
+	NextStage(epoch uint64, passed []ServerInfo) ([]ServerInfo, error)
```

Comment:
The 1st argument should be type of `reputation.Trust` since route may depend not only on the epoch.